src/tools/clippy/tests/ui/macro_use_imports.stderr

```diff
@@ -1,28 +1,28 @@
 error: `macro_use` attributes are no longer needed in the Rust 2018 edition
-  --> $DIR/macro_use_imports.rs:23:5
+  --> $DIR/macro_use_imports.rs:24:5
    |
 LL |     #[macro_use]
-   |     ^^^^^^^^^^^^ help: remove the attribute and import the macro directly, try: `use mac::{inner::foofoo, inner::try_err};`
+   |     ^^^^^^^^^^^^ help: remove the attribute and import the macro directly, try: `use mac::inner::nested::string_add;`
```

Comment:
Why did that change? :thinking:

Reply:
Another one I'm unsure about. The CI also failed, so it might be related to that. I'll try to fix it now.

Reply:
Hm, I think the order of the lint emission just changed. Which is really weird, but doesn't break tests, so I'm ok with it.